src/integrations/mailerlite/class-mailerlite-client.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Mailerlite client implementation
+ *
+ * @package Eightshift_Forms\Integrations\Mailerlite
+ */
+
+declare( strict_types=1 );
+
+namespace Eightshift_Forms\Integrations\Mailerlite;
+
+use Eightshift_Forms\Hooks\Filters;
+use \MailerLiteApi\MailerLite as MailerLiteClient;
+use \GuzzleHttp\Client as GuzzleHttp;
+use \Http\Adapter\Guzzle6\Client as Guzzle6;
+
+/**
+ * Mailerlite integration class.
+ */
+class Mailerlite_Client implements Mailerlite_Client_Interface {
+
+  /**
+   * Mailerlite client object.
+   *
+   * @var MailerLiteClient
+   */
+  protected $client;
```

Comment:
Why `protected`?

Reply:
Not sure, I just copied it from the mailchimp integration. Should it be private?